### DISCUSSION TITLE
Modifying Wings

### DISCUSSION THREAD

**Discussion by FoksVHox** [pterodactyl/panel#2903] [General Community]
Hello!

First off, Pterodactyl in its current state is **awesome**, however, there are some small edit's that I'd like to do the Wings.
So I started with pulling the latest stable version of Wings, I wanted to build at this stage so I was sure the source code worked, all I did beforehand was to change the import paths where the namespace of `pterodactyl` was. 

GoLand is currently reporting the following problems:

- /wings/environment/docker/container.go
    - Line: 176 - "Cannot use 'a.Exposed()' (type "github.com/docker/go-connections/nat".PortSet) as type "github.com/docker/docker/vendor/github.com/docker/go-connections/nat".PortSet
    - Line: 188 - "Cannot use 'a.DockerBindnings()' (type "github.com/docker/go-connections/nat".PortMap) as type "github.com/docker/docker/vendor/github.com/docker/go-connections/nat".PortMap"
    - Line: 227 - "Not enough arguments in call to 'e.client.ContainerCreate'" 

Any ideas on how I can fix this issue?

**Comment by matthewpi** [comment]
Using my dev environment I pulled the latest changes then checked out to the `release/v1.1.3` branch.  I ran `go build ./...` (this will not output a file but will find any errors with the codebase) and it worked properly without any errors.  I'd recommend trying that without changing any.

**Reply by FoksVHox** [reply]
What's your dev environment?

**Reply by matthewpi** [reply]
Not sure exactly what you are asking but hopefully this answers your question.

For the steps I did above those were ran in WSL (Debian) with go1.15.6.  For actually running wings and the panel I use a RHEL8 VM that has the code uploaded to it using Jetbrains' Remote Code Upload (which is available in all their editors).

**Reply by FoksVHox** [reply]
Alright, I'm going to try that

**Reply by FoksVHox** [reply]
@matthewpi It works like a charm using a VM. Thanks
